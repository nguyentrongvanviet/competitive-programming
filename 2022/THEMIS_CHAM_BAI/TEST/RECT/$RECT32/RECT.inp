872
425984023 130565347
-924632536 -778661475
-646692505 531933207
649186662 992417398
519166724 -283121996
71075120 -57659293
236163519 340937371
366300468 -959306868
323791019 637385577
80004756 959798445
882995062 -366605350
694926819 -709928581
-796142312 -127013551
885432179 921175495
-18902017 -450661930
176694653 884392826
103208063 509147686
-993635709 -481306006
-494986107 -243109917
-479584189 591712870
217159763 -754245909
592221067 720921431
-634633016 313057835
-353696341 -125803001
190973191 -660343820
201848454 -750424229
547911725 720933453
115716934 314917839
-706734572 999977953
776971692 798276971
986449771 11675744
-253825016 -910467320
266224315 -761988721
-706734572 999977953
-966143887 176559200
437620895 -627809956
-545706336 922471699
-234524207 306289820
-19767641 184108906
490675018 -390740005
280853136 -264520747
478919862 -49069760
-522245647 763769516
-16232776 -920543246
-725362820 954244543
472042578 637767278
765112934 -113980152
-570480218 65826892
-204281161 7263910
553053321 812267586
-26358034 76690894
-597471606 440488969
-842788256 -458046333
-348683944 71088182
365368992 -742087813
596361790 221033835
-935540237 667238627
-273992880 684596316
755040573 790345768
26991174 -690718745
685540106 -298470262
956411651 -264771839
42200316 538709274
-753269236 -695289309
-225010048 -210156214
200535899 -569906581
-704717099 -133699609
-50838384 215258005
381197519 -979094997
705221360 11054263
302228797 978846200
244650296 -583382997
-312830902 -195722773
641002985 -314545338
437879074 -37403875
-834844609 -518898366
561727635 750934953
-475005809 216688766
202482914 868867704
-252899203 -684740981
-725547131 840877936
-824983093 157463680
-477024924 648452532
668472184 -857915976
-263796445 -788332610
-377369254 -759698241
-613358738 368133440
-781590269 992132505
-419272980 -135643827
-281289078 322184501
736344711 405286915
-112021052 -945634970
-792865540 986805841
288584846 -103959928
45929444 -146823683
334544150 906854824
577605859 -478117133
-832763201 -230161580
545304462 -267533723
-334528346 -318266669
-518743522 -64235719
781686180 -811919756
-603201428 -858421301
-108158979 -170334971
-250516982 352869852
753115632 716241632
477182376 109261966
-619133564 -98945966
703573802 -96352723
663679934 -165491596
195885016 -11590987
-788774924 -601837124
-886660246 -670445018
781686180 -811919756
-337670328 737171850
-289277184 -108745577
-206391141 187658006
848389007 -540494826
961378566 -121048576
88708051 736640636
-525487367 170571123
454623187 942868958
956411651 -264771839
-97164160 -289026222
922750726 -278943674
494404464 -600514392
189265589 133611082
-521432075 -199680025
96881713 237209255
61706671 -933770937
415732128 42665656
-411008783 -549457245
-624434545 949114873
-639115179 608471118
905528740 -803696828
-590644832 -212710941
141143159 243910222
643550267 506695352
471859574 -844776064
-524036414 807353723
-539098876 658895518
-342235822 -358036347
90872577 -325028908
572183629 -467099110
-532199616 -845670339
-76153320 -954470833
8955824 685757796
-189117795 -693432259
393834890 456645527
-885553109 -686934952
-225007693 -222669707
437879074 -37403875
-855807155 27789481
583325478 -263599709
477885519 503981754
771483832 431315908
467319658 -717802646
-851445396 103982818
530506071 -800766149
-385956338 -690926951
160206478 84600670
649609659 564372922
-25452302 -10986957
435766346 -380812037
719454542 990188619
-759614430 -672319673
-251858347 879197999
763834953 499549330
-413485927 996542903
-888525204 -198762506
-530734627 426147715
-296693200 61873531
-494986107 -243109917
985259845 -158430788
-139203742 -422029785
-947589328 806502886
2953241 844034420
-739522862 122140118
-835196569 852398432
411185837 350295100
-483167076 -374100448
917273841 410433454
-585760471 -149126707
-933362977 281334896
-240582388 -297708631
-484459409 -465782892
-862177821 925576959
-15495889 -857732526
-531978714 632013867
494741260 -126088270
480598012 -40328605
922386698 59873173
-562537084 572012287
-263796445 -788332610
-645239988 462438770
590238655 -306815411
986956680 504349567
-524925071 -52082575
-266037212 764258546
502228001 -366660800
824506372 -808471777
205561140 837080533
-962524138 809509856
248581217 -176359419
915907219 -314074821
899373953 149214793
170806861 510087306
735781557 205114871
-874065552 -680725132
763834953 499549330
283123661 819735873
-447877595 -810282695
-36605983 -269929502
-62950200 -904315876
-822672357 234340999
15344879 719606341
881584645 671355969
209608074 910206326
-620511537 -555701274
936193288 124877702
490372337 -188021619
-326928832 -977415186
500213929 -543635846
-248424961 339324239
536389365 110001773
281326489 -588513222
352831524 -523039690
723977637 438807296
176895181 -280373163
-137112776 -849846694
-233015530 202052169
709643811 730645348
-500542416 -982114348
-304408386 -459462479
-539952915 373577156
-441661044 389282269
-657746584 -517520945
624080302 -974683445
590553123 130199640
-802561581 2223455
176487677 -108286456
-28284565 -651958507
846625165 -904585287
-840799574 -743274153
-499099991 -196100169
190148448 994062023
802608106 845590339
411456728 615572955
147450679 -839208467
621066103 352366237
912905203 -83822685
60317721 934362137
804437656 -947678501
-552217770 -922123573
896946055 -740017395
52495206 84062896
-610362843 877507921
536818928 23455527
-90438521 -722215648
339671885 -373073237
986449771 11675744
-76227119 -444826904
-552217770 -922123573
715345048 -572669755
-526273347 169889108
122404015 -159663050
342480023 123175775
827513510 238004285
-657746584 -517520945
14294062 -939444320
74484424 843086149
165374693 769505078
822576301 -4891211
371536082 -323167792
-178520950 632125014
-348410759 -400411676
182788021 262816570
38529190 225522029
345881446 264543514
414401557 -43842252
-60699582 -839548792
656036 -257488840
-874065552 -680725132
-126379682 88056244
-694278886 165565884
-51467937 879494683
76975474 -769712574
-76227119 -444826904
-639049268 652623904
-391720600 -276400553
-149535319 -778452142
-92892623 75536173
-467908355 423778356
257233538 -979105460
36606255 -24520577
-447354874 479715388
213486478 -536001149
636009703 -395483971
-599018470 619942488
51628052 102116435
-316416589 344249406
371536082 -323167792
-250516982 352869852
-401721030 422307700
-626079304 165366869
637378272 -883802681
-131024822 178423930
715216345 577189939
-407870118 -390689303
-205546704 -951662858
-923691225 874203989
407944718 -625764869
-858431652 68846443
-733185493 -576769932
781554443 488109278
-52508609 266229094
-12774907 -512216981
-576682469 427587475
-646692505 531933207
736344711 405286915
-103669737 565505343
-820743399 -79608211
229613063 72440137
572183884 -484514627
-383175385 557629858
709483341 -93898787
720106333 855714990
-382018938 792957099
460256412 -682326225
-250638251 679249650
-585091341 -318174505
-57660100 -526899396
217159763 -754245909
165777421 172953797
-677811612 -600589599
76779865 92742162
720122773 612431943
-115423831 799528774
93115898 110837651
64581078 -561540201
668147789 -25791166
-868357363 -798195801
-366909201 -333328623
-282067531 -716998021
517682798 -4558067
-523079336 -974299487
478919862 -49069760
-666823534 -271531587
208759505 423018764
694504953 -934348035
166359751 232659946
-533388551 740197140
189843419 769893412
-474619342 -278284478
-439874380 -417247681
515266550 -250948008
-274048426 -433877658
-899001728 922792616
-742777104 65719583
-470204336 862019697
-645456488 394652309
-260163825 440048095
930789064 967999904
141143159 243910222
-413485927 996542903
22811458 528279142
-853390324 96900217
243620162 212191489
449655728 447566590
866531489 -797755295
-414903328 -214846186
115028083 368343268
179352070 -971765760
-901601760 -883266002
446545837 -548947266
-534563462 -517082625
544245750 144856378
645418376 -300087822
-76153320 -954470833
-402303615 -390417989
-5616720 -623006877
-828392101 -419957110
641002985 -314545338
-574292562 896921074
-863740321 -203145673
599722044 723044930
146125087 -6018462
848117623 562799802
939456021 -25326329
-543777944 -440458654
760619171 30975239
-328625568 85976687
-112981424 -786030141
616016237 796891261
-739522862 122140118
302228797 978846200
-933362977 281334896
-901601760 -883266002
550771587 200065744
798070230 518382433
6773740 -27735018
903358171 -385230780
-749874271 -425548004
364063335 498334554
-377369254 -759698241
-814118803 -638660952
407434340 -321232083
-420625813 929686146
-380714882 -337046759
326258214 87619805
-44796905 -323242805
-507658035 -695094171
-56716230 -297919744
623937803 -705432557
623068500 -310180188
-126180400 826326724
-155868856 598368530
291647722 41502622
853448469 23056093
-417759210 -810889519
-232453604 734485512
918768984 347245423
-619133564 -98945966
400207724 -362739105
530163458 344854501
490675018 -390740005
-495893911 -19349194
-426523528 211080882
-861900830 -287333472
-634613118 839188672
-68018941 -875143373
760619171 30975239
816303752 -419410917
136774263 704819547
283123661 819735873
645650775 -233837495
-537941526 96006273
-559361834 766692244
819439169 -233877767
-446093163 -657514254
457497340 247947917
-35043995 91754841
359839163 -126192490
-606532700 197006009
653715735 843877278
154139839 169320483
-388415332 -160026682
95363019 692162258
830572968 -884934084
-500100960 208959185
-583839541 -796416841
471974042 -729179482
488642254 274906873
989405771 -966802482
-209589487 -107409815
592863998 -64942871
-240582388 -297708631
415877544 456056072
267160219 588760021
857013735 -268154023
-246855624 924675186
38529190 225522029
600351008 -400644441
753564380 523014609
-669372572 155345298
336691615 -588026354
699885990 145088320
-304408386 -459462479
636333843 -857790896
144188615 650201450
-275188514 272881095
205054709 991508335
-832763201 -230161580
-488699238 -248401893
959287695 -439378810
-467439244 -484717482
-780475925 -407959394
441570640 61632115
-418270803 -135791816
-370223686 912419132
-710874038 324548232
989780170 -573838209
-349691450 -889209527
496490801 -564530001
6446046 -268353052
-124606837 -70935630
552190661 717038101
-157831222 -353348343
100931266 -429012746
701054242 603854206
660727912 -499329474
-352908751 -191544678
-444577113 -366621409
-17807950 686132875
-492151726 483265929
21078898 993623537
116764697 -29861234
117982954 -678160387
790307872 775161586
369010197 9162637
-273992880 684596316
980594549 271931402
-509565989 714623712
707891106 524448792
752305834 -400946059
928703777 489593201
-218541223 -78255706
319298954 384064821
-279498391 -706069935
-947589328 806502886
903701392 279783833
431813916 777296284
-226876956 478225221
58130601 941883484
344680642 855340108
-727779708 16413868
-251858347 879197999
-447354874 479715388
-342478680 -665119200
186816102 572444957
82594628 463994611
369975737 -61387899
-762921711 744435026
658464233 -888606507
671894300 -118972799
628575024 -750615168
480598012 -40328605
61706671 -933770937
-840799574 -743274153
-10763533 670491000
-597263235 -161161156
222959590 316478121
-792865540 986805841
-944564051 536762286
-663840296 322015797
-700013835 968702456
-159846059 725309870
317374177 76510792
-74837220 -194193332
-727779708 16413868
-598875005 410332631
952536811 -70870455
391122972 -940846857
-441451963 -630061490
-873574865 -467012905
-644862507 236583929
86993025 -399297419
422209191 -20029
-563441134 -251079153
-137541794 -319848677
4619558 -719687249
787266113 -715058572
-347641208 -76706990
990998478 -301224932
-278173165 387025596
231147972 -108743258
-779403772 339538106
111528586 210789598
146261226 -733388820
321150302 360204010
18165085 146654181
-170796115 -614417765
-855890940 -442906183
-69350527 -640788187
820537992 -636811377
-680762166 -693447045
54210155 -734797147
-307066476 660119232
624080302 -974683445
-716962396 -265906555
337406370 770179774
-460302754 -485586085
41535318 -712669265
-873490747 -829659155
134203934 35064076
81673002 537990370
941552955 -420903043
778865299 -255369598
965471204 -946363317
-770533622 31482574
-7866902 533035677
676882831 -218311066
22811458 528279142
371234770 -287086346
271526503 -252203476
-411008783 -549457245
-402986247 -372702707
-20261153 -503615047
133042883 583697994
296826239 -825280008
312402440 92054977
-910071635 5736248
975469872 982602768
586888638 -814880443
-423827500 454357756
208759505 423018764
923481743 517140962
-952750009 -497398997
-338030427 -688102868
-528932521 -304796889
-47928461 -8482682
682961290 456063646
667696356 889995132
626788277 854584107
707359092 -534609736
899373953 149214793
422518212 778136246
530255708 120978850
-770033545 -372747840
369010197 9162637
-35043995 91754841
519040565 -131149067
-121682151 -331246212
-404605288 -646846874
-918212299 611987147
974472899 -433921823
473318691 60997036
676471777 -227568869
-178520950 632125014
-580427414 -385867789
918703312 73894228
-641278942 461309878
-935020943 -6247146
918703312 73894228
-873932849 22584728
-424174176 53475721
-79280950 207350722
-846918799 842857743
-643401332 488047919
-709445829 -910027066
-254859027 309059878
850782222 -489771323
81442725 515347260
189843419 769893412
359388141 -554889540
-853604268 -890268566
699885990 145088320
48932773 328535505
471964755 -536704254
-37958816 614367026
108297294 153289793
-3091984 -98045074
431813916 777296284
-515945382 -646274880
809367213 56912548
42200316 538709274
75945236 883965341
989780170 -573838209
-36785199 -648482917
129307010 -797916638
-738625370 -81000792
692574247 -795567101
-227327002 989970697
895680291 -246766354
75945236 883965341
117745716 -241939390
-60699582 -839548792
-13571498 -77549090
440408203 -132538322
-274048426 -433877658
-472981030 523906315
-762921711 744435026
-650248916 -517823985
488087176 -266991657
29572653 344026065
285818763 -616785976
-438493342 401028757
-950814542 746635471
-447028072 -479570013
819439617 32140062
926725765 -486856318
397142054 -533036835
-125944096 147235531
929428623 -874259665
452656963 -260123899
435649426 742711867
860174648 242384635
415500363 -308857302
471964755 -536704254
556766077 465260239
81548079 168284044
-979791042 -295010010
830572968 -884934084
-500805411 141545146
-355025117 -148271143
-459123944 -644274899
-811864358 588739476
-855807155 27789481
-920925514 963729596
-545078273 -858564314
-114634936 240258603
709643811 730645348
-205546704 -951662858
261986390 233222409
-558309578 698099237
-863222110 670246988
930789064 967999904
95363019 692162258
926761139 -572473814
-79659669 -32170888
221351584 -783110982
-95508334 241007166
955898057 -759110465
535577334 -679674561
-267110073 601818427
775534360 308458997
-899967888 -824112336
370917588 -938605422
-966143887 176559200
164585173 569595519
553053321 812267586
822576301 -4891211
526514531 -11057276
-450730595 158778521
905528740 -803696828
-555796532 -705744347
720137092 437179419
-89938939 -46547603
-381911352 -744461200
848472531 -49035481
13985663 49516590
583807343 142780601
-60426296 -192229754
703573802 -96352723
-500100960 208959185
-41324500 -824754970
-674678818 304021644
850148591 412614089
136774263 704819547
505525038 -186847461
-328242700 51188180
-697356887 -422421330
830944254 -699376231
-891403004 856980678
-414903328 -214846186
291631903 -434484951
709879401 216426632
-894454913 -495774619
820537992 -636811377
103208063 509147686
-470204336 862019697
254168730 -133575011
-153395492 532630895
-576682469 427587475
516341971 29783634
500781718 -111202923
359388141 -554889540
-25452302 -10986957
438070557 -736765765
-267110073 601818427
676882831 -218311066
-164570947 -532983588
-167042451 796671886
627191298 748403054
-196619285 652007858
-380714882 -337046759
373717815 555299746
888117193 -645538165
-555796532 -705744347
-596377009 -577368718
261655471 -374147541
549801640 -529495075
773042425 -45580803
-645456488 394652309
732386337 49335613
-613364279 703383677
-142470031 -642401415
64052159 -665337628
240342860 388196388
884009161 526233936
117701326 -597462070
655023936 -661500293
891144959 -631467217
-307705644 -693117015
86993025 -399297419
756028630 400613553
397142054 -533036835
831293045 -69026096
210718661 53169224
166359751 232659946
908114812 354495463
-634633016 313057835
-13507198 769955830
-802306536 738043204
169648558 -398514582
-457245657 57621527
173695612 9830724
-360341262 736387335
443019462 -988432147
-645186680 -354938792
616016237 796891261
-583670625 -141211814
85888179 566494633
-770033545 -372747840
82802637 519790729
-3255525 -977123422
-425133138 735659077
-489554475 694248906
513935716 -996405704
476147302 863012918
-863222110 670246988
-533388551 740197140
425984023 130565347
-234175809 -175186400
272722883 148306526
424546446 -502684257
526741641 -82799091
-257383450 779315844
-436070505 -571426277
-899001728 922792616
8955824 685757796
-540976622 -192077202
-329440894 206972411
94475322 -981999409
989094407 -102755749
-663108551 -248189727
-60426296 -192229754
-556788748 -502916688
-755115235 101665713
102412392 235224752
-550245678 -120226743
689912664 -766167875
-107029512 -785101915
152598714 -125189319
297037432 835266167
-492151726 483265929
959287695 -439378810
-482399808 128638163
120504754 227637932
444691849 -796022135
355522172 -302250430
918549030 774918515
115028083 368343268
-554813405 -179401240
-450730595 158778521
496490801 -564530001
645650775 -233837495
790524854 140321193
341586274 77108620
-400024940 -605969697
-527443527 -463428169
-579413134 536198891
-12492239 304127655
-597662789 -458275082
516341971 29783634
-390199710 51733578
-618459266 -987896542
907457522 -355259477
681870131 944732514
-558735007 510546637
137209675 164967796
-21191009 948042954
332737598 780410827
-893152473 -780909967
130087812 -118176275
-401542607 -749479347
261734112 -294008807
-109302388 -941120214
745857567 -80118636
-597471606 440488969
802608106 845590339
-879305479 392689266
424322143 434684982
547911725 720933453
864808081 200022845
749473075 -665587543
66373317 -220719670
-439874380 -417247681
457759697 376917704
756628131 113584889
486650730 -798040280
407877529 -547753183
