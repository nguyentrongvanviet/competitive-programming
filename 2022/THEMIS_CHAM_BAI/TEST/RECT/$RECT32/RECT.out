1203529220946365833